src/main/java/net/openhft/chronicle/bytes/Bytes.java

```diff
@@ -62,9 +62,8 @@
  * Generally, a Bytes object is a resource that is ReferenceCounted and certain operations invoked on a Bytes
  * object that has been released, might result in an {@link IllegalStateException} being thrown.
  *
- * @see BytesStore
  * @param <U> Underlying type
- *
+ * @see BytesStore
```

Comment:
@see does usually go before @param's

Reply:
I think the order as it is now complies with https://www.oracle.com/technical-resources/articles/java/javadoc-tool.html#orderoftags